code/processes/dqe.q

```diff
@@ -5,6 +5,7 @@ configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqeconfig.csv"]];
 init:{
   .lg.o[`init;"searching for servers"];
   .servers.startup[];                                                                                           /- Open connection to discovery
+  .proc.loaddir[(getenv `TORQHOME),"/code/dqe"];                                                                /- load in dqe function code directory
```

Comment:
as long as configuration is right (I think `.proc.loadprocesscode` needs set to `1b`), this should happen automatically